test/integration/retryable-reads/retryable_reads.spec.test.js

```diff
@@ -28,3 +37,87 @@ describe('Retryable Reads', function () {
     );
   });
 });
+
+// These tests are skipped because the driver 1) executes a ping when connecting to
+// an authenticated server and 2) command monitoring is at the connection level so
+// when the handshake fails no command started event is emitted.
+const SKIP = [
+  'find succeeds after retryable handshake network error',
+  'find succeeds after retryable handshake network error (ShutdownInProgress)'
+];
+
+describe('Retryable Reads (unified)', function () {
+  runUnifiedSuite(loadSpecTests(path.join('retryable-reads', 'unified')), SKIP);
+});
+
+describe('Retryable Reads', function () {
+  const dbName = 'retryable-handshake-tests';
+  const collName = 'coll';
+  const docs = [
+    { _id: 1, x: 11 },
+    { _id: 2, x: 22 },
+    { _id: 3, x: 33 }
+  ];
+  let client;
+  let db;
+  let coll;
+
+  beforeEach(function () {
+    if (
+      semver.lt(this.configuration.buildInfo.version, '4.2.0') ||
```

Comment:
should there be an auth requirement here?

Reply:
Added auth and serverless checks.

Reply:
This one shouldn't need a serverless check according to the unified spec test file; and we can probably do the other checks via the test metadata so that we don't have to manually do the skipReason

Reply:
updated this logic